### PR TITLE
fix(release-please): uses factory from latest version

### PR DESCRIPTION
**Pull request by pragmaticivan**
* uses factory from latest version
* use release-please@next
* fixes tests with factory usage

### CODE REVIEW

**Comment by pragmaticivan** [comment]
Tested it here: https://github.com/remoteday/rd-api/releases

* Release PR: https://github.com/remoteday/rd-api/runs/1899038891
* Release Tag: https://github.com/remoteday/rd-api/runs/1899041119
* Commit SHA: https://github.com/remoteday/rd-api/blob/main/.github/workflows/release-please.yml#L10

**Comment by pragmaticivan** [comment]
It's currently using @next so if you want to merge it, I can just put the latest workable version in v10 so it would fix @v2.

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "homepage": "https://github.com/bcoe/release-please-action#readme",
   "dependencies": {
     "@actions/core": "^1.2.6",
-    "release-please": "^10.1.0"
+    "release-please": "next"
```
Comment (bcoe) [inline package.json:29]:
```suggestion
    "release-please": "11.0.0-candidate.1"
```

**Review by bcoe — APPROVED** [review]
Thanks for the fix 👍 

I've already rolled back the v2 tag to the last good release, so it should be good (It's want created the open release PR for release-please).

I'm going to follow on with a couple additional tests before we release.

**Pull request merged**